INSTRUCTION.md

```diff
@@ -0,0 +1,19 @@
+## How to validate App is running
+``
+     http://localhost:30007
+``
+
+## How to validate configMap data is mounted as files in the right order
+## Secret data is mounted as a file
+
+``
+
+    kubectl exec kube2py-856b5b48f5-4zq6f -it -- sh 
```

Comment:
The pod name 'kube2py-856b5b48f5-4zq6f' seems to be hardcoded. Consider providing instructions on how to dynamically retrieve the pod name, for example, using a label selector with 'kubectl get pods'.